### PR TITLE
Merge all samples into one gradle module and app.

### PR DESCRIPTION
**Pull request by zach-klippenstein**
Three atomic commits:
1. Merge all samples into one gradle module and app.
2. Make sample launcher animate sample activities from their list items.
3. Show previews of each sample in the list.

Screenshot_1590950927

### CODE REVIEW

File: core-compose/src/main/java/com/squareup/workflow/ui/compose/internal/ParentComposition.kt [hunk 1]
```diff
@@ -17,24 +17,19 @@
...
-import androidx.ui.core.setContent
 import com.squareup.workflow.ui.ViewEnvironment
 import com.squareup.workflow.ui.ViewEnvironmentKey
 
 /**
- * Holds a [CompositionReference] and that can be passed to [setOrSubcomposeContent] to create a
- * composition that is a child of another composition. Subcompositions get ambients and other
- * compose context from their parent, and propagate invalidations, which allows ambients provided
- * around a [WorkflowRendering] call to be read by nested Compose-based view factories.
+ * Holds a [CompositionReference] and that can be passed to [setContent] to create a composition
```
Comment (rjrjr) [inline core-compose/src/main/java/com/squareup/workflow/ui/compose/internal/ParentComposition.kt:25]: s/and//

File: samples/src/main/java/com/squareup/sample/launcher/SampleLauncherApp.kt [hunk 2]
```diff
@@ -56,7 +62,41 @@ import com.squareup.sample.R
...
+
+private fun launchSample(
+  sample: Sample,
+  rootView: View,
+  sourceBounds: PxBounds?
+) {
+  val context = rootView.context
+  val intent = Intent(context, sample.activityClass.java)
+  val options: Bundle? = sourceBounds?.let {
+    makeScaleUpAnimation(
```
Comment (rjrjr) [inline samples/src/main/java/com/squareup/sample/launcher/SampleLauncherApp.kt:149]: Now you're just showing off.

File: samples/src/main/java/com/squareup/sample/launcher/SampleLauncherApp.kt [hunk 3]
```diff
@@ -74,14 +86,58 @@ import com.squareup.sample.R
...
+      secondaryText = { Text(sample.description) },
       singleLineSecondaryText = false,
-      modifier = Modifier.onPositioned { globalBounds.value = it.globalBounds },
+      // Animate the activities as scaling up from where the preview is drawn.
+      icon = { SamplePreview(sample) { globalBounds.value = it.globalBounds } },
       onClick = { launchSample(sample, rootView, globalBounds.value) }
   )
 }
 
+@Composable private fun SamplePreview(
```
Comment (rjrjr) [inline samples/src/main/java/com/squareup/sample/launcher/SampleLauncherApp.kt:98]: 🤯

**Comment by rjrjr** [comment]
giphy

File: samples/src/main/java/com/squareup/sample/launcher/SampleLauncherApp.kt (hunk 2 above)
Comment (zach-klippenstein) [reply samples/src/main/java/com/squareup/sample/launcher/SampleLauncherApp.kt:149]: Couldn't help myself. It's like a real launcher! 😅

**Pull request merged**